processes/dags/pubmed_migration.py

```diff
@@ -105,11 +104,11 @@ def make_tarfile(output_filepath: str, source_dir: str) -> None:
         tar.add(source_dir, arcname=os.path.basename(source_dir))
 
 
-def delete_temp() -> None:
+def delete_temp(filename: str) -> None:
```

Comment:
What is this doing? Is it deleting a file or a directory? If its a directory, the function appears to be redundant with delete_dir.